docs/patterns/components/Tabs/index.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { Tabs, Tab, TabPanel, Code } from '@deque/cauldron-react';
+import './index.css';
+
+const Demo = () => {
+  const [value, setValue] = useState(0);
+  const handleChange = newValue => setValue(newValue);
+
+  return (
+    <div className="TabsDemo">
+      <h1>Tabs</h1>
+      <h2>Demo</h2>
+      <h3>Baisc Tabs</h3>
+      <Tabs value={value} handleChange={handleChange}>
+        <Tab index={0} value={value}>
```

Comment:
The `index` seems a bit kludgy, perhaps using a `ref` would be better here?

```jsx
<Tabs>
  <Tab target={panel1}>1</Tab>
  <Tab target={panel2}>2</Tab>
  <TabPanel ref={panel1}>1</TabPanel>
  <TabPanel ref={panel2}>2</TabPanel>
</Tabs>
```

We could also default to the element order, so that the tabs/tab panels will match their index unless otherwise specified:

```jsx
<Tabs>
  <Tab>1</Tab>
  <Tab>2</Tab>
  <TabPanel>1</TabPanel>
  <TabPanel>2</TabPanel>
</Tabs>
```

Reply:
For the second example, we could still use `refs` behind the scenes.